.github/workflows/tagPublish.yml

```diff
@@ -3,7 +3,7 @@ name: packaging
 on:
   push:
     tags:
-      - "*"
+      - '[0-9]+.[0-9]+.[0-9]+' # publish only on a release tag and not on RC candidates.
```

Comment:
I think we should do it on the release candidates. That will detect problems sooner.

Reply:
ok, fixed it.